eldritch/mythos.py

```diff
@@ -227,6 +319,80 @@ def create_event(self, state):
     return seq
 
 
+class Mythos12(Headline):
+  def __init__(self):
+    super().__init__("Mythos12", "Square", "Unnamable", {"circle"}, {"square", "diamond"})
+
+  def create_event(self, state):
+    seq = super().create_event(state)
+    seq.events.append(events.ReturnToCup(
+        from_places={"University", "Library", "Administration", "Science"}
+    ))
+    return seq
+
+
+class Mythos13(Rumor):
+  def __init__(self):
+    super().__init__(
+        "Mythos13", "Cave", {"slash", "triangle", "star"}, {"hex"}, "Rivertown"
+    )
+
+  def should_fail(self, state):
+    return state.terror >= 10
+
+  def get_interrupt(self, event, state):
+    if not self.failed and isinstance(event, events.EncounterPhase):
+      return self.get_pass_interrupt(event, state)
+    return None
+
+  def get_pass_interrupt(self, event, state):
+    if event.character.place != state.places["Rivertown"]:
+      return None
+    seq = events.Sequence([
+        events.EndRumor(self, failed=False),
+    ] + [events.Draw(char, "spells", 1) for char in state.characters if not char.gone])
+    return events.BinarySpend(
+        event.character, "gates", 2, "Spend 2 gate trophies to end the rumor?",
+        "Yes", "No", seq
+    )
+
+  def get_trigger(self, event, state):
+    if isinstance(event, events.IncreaseTerror) and self.should_fail(state):
+      curses = [events.Curse(char) for char in state.characters if not char.gone]
+      return events.Sequence(curses + [events.EndRumor(self, failed=True)])
+    return super().get_trigger(event, state)
+
+  def progress_event(self, state):
+    first_player = state.characters[state.first_player]
+    dice1 = events.DiceRoll(first_player, 1)
+    prog1 = events.IncreaseTerror()
+    cond1 = events.Conditional(first_player, values.Die(dice1), "", {0: prog1, 3: events.Nothing()})
+    return events.Sequence([dice1, cond1])
+
+  def get_progress(self, state):
+    return state.terror
+
+  def max_progress(self, state):
+    return 10
+
+
+class Mythos14(Environment):
+  def __init__(self):
+    super().__init__(
+        "Mythos14", "Unnamable", "Woods", {"square", "diamond"}, {"circle"}, "urban", "Northside"
+    )
+
+  def get_trigger(self, event, state):
+    if isinstance(event, events.Movement) and event.character.place.name == "Northside":
+      clues = events.Gain(event.character, {"clues": 1})
+      check = events.Check(event.character, "will", -1)
+      loss = events.Loss(event.character, {"sanity": 1})
+      return events.Sequence(
+          [clues, events.PassFail(event.character, check, events.Nothing(), loss)]
+      )
+    return None
+
+
```

Comment:
These mythos cards will be significantly more effective if you include them in CreateMythos().